README.md

```diff
@@ -134,6 +146,36 @@ profile data was requested, the response's `profile` key will store a dictionary
 | `message`   | `str`           | A message that provides information corresponding to the status          |
 | `timestamp` | `datetime`      | A timezone offset timestamp indicating the time of authentication        |
 
+
+### `/health` endpoint
+
+This endpoint can be used to check the health of the API. It's useful for monitoring and uptime checks.
+
+### Request Parameters
+
+This endpoint does not take any request parameters.
+
+### Response Object
+
+| **Field** | **Type** | **Description**                        |
+|-----------|----------|----------------------------------------|
+| `status`  | `str`    | The status of the API, typically "ok". |
+
+
+### `/readme` endpoint
+
+This endpoint redirects to the project's official GitHub repository.
+
+
+### Request Parameters
+
+This endpoint does not take any request parameters.
+
+### Response Object
+
+This endpoint returns a `307 Temporary Redirect` to the GitHub repository.
+
+
```

Comment:
The endpoint does not return anything. There is no response object here since it redirects. You do not need these sections. I would say just add this line after "This endpoint redirects to the project's official GitHub repository." that says: `This endpoint does not take any request parameters.`